### PR TITLE
lots of small cleanups

### PR DESCRIPTION
**Pull request by ged-odoo**

### CODE REVIEW

File: addons/wowl/static/src/main.ts [hunk 1]
```diff
@@ -20,13 +20,20 @@ declare const odoo: Odoo;
 
   // load templates
   const templatesUrl = `/wowl/templates/${odoo.session_info.qweb}`;
-  const [templates, { localizationParameters, _t }] = await Promise.all([
+  const [templates, { localization: localizationParameters, _t }] = await Promise.all([
```
Comment (Polymorphe57) [inline addons/wowl/static/src/main.ts]:
```suggestion
  const [templates, { localization, _t }] = await Promise.all([
```

File: addons/wowl/static/src/main.ts [hunk 2]
```diff
@@ -20,13 +20,20 @@ declare const odoo: Odoo;
...
     loadFile(templatesUrl),
-    fetchLocalizationParameters(browser, odoo),
+    fetchLocalization(browser, odoo),
   ]);
 
   // setup environment
-  const env = await makeEnv({ browser, localizationParameters, odoo, registries, templates, _t });
+  const env = await makeEnv({
+    browser,
+    localization: localizationParameters,
```
Comment (Polymorphe57) [inline addons/wowl/static/src/main.ts]:
```suggestion
    localization,
```

File: addons/wowl/static/src/core/hooks.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import { Component } from "@odoo/owl";
+import { OdooEnv, Services } from "../types";
+
+// -----------------------------------------------------------------------------
+// Hook and deploy function
```
Comment (aab-odoo) [inline addons/wowl/static/src/core/hooks.ts]: and deploy?

File: addons/wowl/static/src/types.ts [hunk 4]
```diff
@@ -54,3 +70,58 @@ export interface RuntimeOdoo {
...
+
+export interface OdooEnv extends Env {
+  browser: OdooBrowser;
+  services: Services;
+  registries: Registries;
+  bus: EventBus;
+  _t: (str: string) => string;
+}
+
+export interface Registries {
```
Comment (aab-odoo) [inline addons/wowl/static/src/types.ts]: still necessary?

File: addons/wowl/static/src/types.ts [hunk 5]
```diff
@@ -54,3 +70,58 @@ export interface RuntimeOdoo {
...
+  browser: OdooBrowser;
+  localization: Localization;
+  odoo: Odoo;
+  templates: string;
+  _t: (str: string) => string;
+}
+
+// -----------------------------------------------------------------------------
+// Types
+// -----------------------------------------------------------------------------
```
Comment (aab-odoo) [inline addons/wowl/static/src/types.ts:115]: weird comment

**Review by aab-odoo — APPROVED** [review]

**Pull request merged**